go.mod

```diff
@@ -29,11 +29,11 @@ require (
 	k8s.io/client-go v0.31.0
 	knative.dev/eventing v0.44.0
 	knative.dev/eventing-kafka-broker v0.37.0
-	knative.dev/hack v0.0.0-20250117112405-6cb0feb3ac46
+	knative.dev/hack v0.0.0-20250220110655-b5e4ff820460
 	knative.dev/networking v0.0.0-20241022012959-60e29ff520dc
-	knative.dev/operator v0.43.1
+	knative.dev/operator v0.43.3-0.20250312021337-6b0f076ba7ae
```

Comment:
did you get this via `make generated-files` and its call to update deps?

Reply:
these 2:
```
go get -u knative.dev/operator@release-1.16
make generated-files
```